torch/optim/adagrad.py

```diff
@@ -91,7 +91,7 @@ def make_sparse(values):
                     state['sum'].add_(make_sparse(grad_values.pow(2)))
                     std = state['sum'].sparse_mask(grad)
                     std_values = std._values().sqrt_().add_(group['eps'])
-                    p.data.add_(-clr, make_sparse(grad_values / std_values))
+                    p.data.add_(make_sparse(grad_values / std_values), alpha=-clr)
                 else:
                     state['sum'].addcmul_(1, grad, grad)
```

Comment:
`addcmul_` should also be raising a deprecation warning, so it might be a good idea to collect and fix all deprecated calls.

Reply:
Sure, let me also check other deprecated calls.